src/DurableTask.AzureStorage/AzureStorageOrchestrationService.cs

```
@@ -1244,6 +1244,12 @@ public async Task CreateTaskOrchestrationAsync(TaskMessage creationMessage, Orch
                 return;
             }
 
+            if (existingInstance != null && !OverrideConditionsMet(existingInstance.OrchestrationStatus))
+            {
+                // An instance in a state that is configured to not be overridable already exists
+                return;
```

I think we should throw an `InvalidOperationException` in this case. Otherwise customers might not realize that their attempt to start an orchestration was ignored.